modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/brainparcellationmesh/v0.1.0.json

```diff
@@ -0,0 +1,66 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/atlas/brainparcellationmesh/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "import": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0",
+    "{{base}}/schemas/neurosciencegraph/atlas/brainparcellationmesh/v0.1.0",
+    "{{base}}/schemas/nexus/core/distribution/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:BrainParcellationMeshShape",
+      "@type": "sh:NodeShape",
+      "label": "This shape is for describing morphology release.",
+      "targetClass": "nsg:BrainParcellationMesh",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "schema:distribution",
+              "name": "Distribution",
+              "description": "Brain parcellation mesh location.",
+              "node": "this:BrainParcellationMeshDistributionShape",
+              "minCount": 1
```

Comment:
Will the brain parcellation mesh stored in Nexus or not ? It depends on its size I guess. But if it is stored in Nexus then the schema:distribution can't be mandatory because data is attached to a given entity after the netity is registered in Nexus.